.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
```

Comment:
Can you add explicit separate entries for the backend and frontend directories?

Reply:
I have done that.